src/dynamic_graph_manager/dgm_solo12.cpp

```diff
@@ -147,6 +147,18 @@ void DGMSolo12::get_sensors_to_map(dynamic_graph_manager::VectorDGMap& map)
         map_motor_board_enabled[i] = motor_board_enabled[i];
         map_motor_board_errors[i] = motor_board_errors[i];
     }
+
+    /**
+     * Connection quality
+     */
+    map.at("sent_commands_messages")[0] =
+        solo_.get_master_board_interface()->GetCmdSent();
+    map.at("lost_commands_messages")[0] =
+        solo_.get_master_board_interface()->GetCmdLost();
```

Comment:
For me it reads easier with only a single plural form, so dropping the `s` after command:

```
sent_command_messages
lost_command_messages
```

Reply:
I fully, agree, I'm actually not sure why I chose those names (I thought I adopted them from some lower level but cannot find it anywhere now...).
I would also change "messages" to "packets" to be more consistent with the naming in MasterBoardInterface.

Reply:
done